packet.go

```diff
@@ -10,16 +10,21 @@ type Packet interface {
 	Unmarshal(rawPacket []byte) error
 }
 
-// Unmarshal is a factory a polymorphic RTCP packet, and its header,
-func Unmarshal(rawPacket []byte) (Packet, error) {
+// unmarshal is a factory which pulls the first RTCP packet from a bytestream,
+// and returns it's parsed representation, and the amount of data that was processed.
+func unmarshal(rawData []byte) (Packet, int, error) {
```

Comment:
Maybe add named returns for readability? Not clear what that int represents.

```go
func unmarshal(rawData []byte) (packet Packet, written int, err error)
```

Reply:
I totally agree with this, done.